class UserNotFound(Exception):
    pass

class RecordNotFound(Exception):
    pass